src/common/src/lru.rs

```diff
@@ -176,6 +176,39 @@ where
         }
     }
 
+    pub fn remove(&mut self, key: &K) -> Option<V> {
+        unsafe {
+            let hash = self.hash_builder.hash_one(key);
+
+            if let Some(ptr) = self.map.find(hash, |p| p.as_ref().key() == key) {
+                let ptr = *ptr;
+
+                // Detach the entry from the LRU list
+                self.detach(ptr);
+
+                // Remove entry from the hash table
+                match self
+                    .map
+                    .entry(hash, |p| p.as_ref().key() == key, |p| p.as_ref().hash)
+                {
+                    Entry::Occupied(o) => {
+                        o.remove();
+                    }
+                    Entry::Vacant(_) => {}
+                }
```

Comment:
Is it possible to replace `entry()` and `find()` with `entry()` only here?

Reply:
Good catch. Done.